giraffe/src/components/PlotResizer.tsx

```diff
@@ -28,16 +29,16 @@ export const PlotResizer: FC<PlotResizerProps> = props => {
     config.staticLegend
   )
 
-  const sizedConfig: SizedConfig = useMemo(
+  const normalizedConfig: SizedConfig = useMemo(
     () => ({
-      ...config,
+      ...normalizeConfig(config),
       height: resized.height,
       width: resized.width,
     }),
-    [config, height, width]
+    [config, height, width] // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
No, we can't put `normalizedConfig` into the dependency array, because this hook is the function that initializes `normalizedConfig`.

What's happening here is this: "if the user's sends in an updated configuration, or if the height and/or width the Plot changes, calculate a new normalized config"